README.md

```diff
@@ -59,6 +59,28 @@ from other services too: you'll just need to provide your file URLs for import.
                                   check requests.
                                   
   ```              
+ 
+## Output file format
+The output file format is very simple.
+For every file URL or Filestack file handle there is 3 columns divided by '\t'(tab).
+
+First column - URL of Filestack handle iteself. 
+
+Second column - upload operation status - `succes` or `failed`.
+
+Third column - The Uploadcare URL of uploaded file or error description.
```

Comment:
The first column holds input file URL or Filestack handle itself, second column — upload operation status: `success` or `failed`, third column — output Uploadcare URL or error description.